_posts/2016-10-16-how-i-removed-two-databases-with-one-pull-request.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How I removed two databases in one pull request
+authors: tge
+---
+
+My first project right after school was called KISS which means
```

Comment:
Maybe you should add more details about your project. Was it an academic paper on KISS, a product or service where KISS influenced the architecture design, the refactoring, etc.? What did you learn in that project that's important for you to mention here?